API.md

```diff
@@ -0,0 +1,49 @@
+<pre>Activities</pre>
+title | CharField  
+subtitle | CharField  
```

Comment:
Right now in each Module for each activities they have subtitiles, but if you think some might not have them then I can totally make this optional.